### DISCUSSION TITLE
Sir, Please try to help me sir I want to open firefox with a vpn

### DISCUSSION THREAD

**Discussion by Okkbro501** [jlesage/docker-firefox#211] [Q&A]
**Sir**
I want that my firefox open with a already connected vpn
Please answer how to do that
Please please please...

I hope you solve my problem

**Comment by jlesage** [comment]
I'm not personally using a such setup, but the idea is to run a Docker container (e.g. `gluetun`) providing the VPN connection and then connecting Firefox's container to the VPN container's network.

You should be able to find multiple tutorials showing how to do this.  Here is one example: https://xtom.com/blog/how-to-route-your-docker-containers-through-a-vpn-gluetun-guide/
